packages/types/ContactJob.ts

```diff
@@ -43,7 +43,7 @@ export type PublishToContactJobsTopicParams = PublishRetrieveContactTranscript;
 type CompletedContactJobMessageCommons<TSuccess, TFailure> =
   | {
       attemptResult: 'success';
-      attemptPayload: TSuccess;
+      attemptPayload: { bucket: TSuccess; key: TSuccess };
```

Comment:
TSuccess is type `string` because this is set at call time. It may be different between transcript and recordings which will come later which is why it is set this way.

See line 53:54 for how it is being set:

```typescript
export type CompletedRetrieveContactTranscript = PublishRetrieveContactTranscript &
  CompletedContactJobMessageCommons<string, any>;
```

Modifying the abstracted attemptPayload breaks the abstraction.

Reply:
@robert-bo-davis If I should refactor, it would look like this:

`attemptPayload: TSuccess;`

```
export type CompletedRetrieveContactTranscript = PublishRetrieveContactTranscript &
  CompletedContactJobMessageCommons<object, any>;
```
Is this what you mean?